main.rb

```diff
@@ -29,17 +30,61 @@ def choose_number(choice, app)
 end
 
 def main
-  app = App.new
+  # load the data and then parse it to the app
 
+  books = load_book_data
+  persons = load_persons_data
+  rentals = load_rental_data(persons, books)
```

Comment:
- [ ] I noticed your method name is load_rental_date but you used it as load_rental_data, kindly rename this too.